test_msgs/action/FibonacciGoalResult.srv

```diff
@@ -0,0 +1,6 @@
+# service respose fields are the same for all actions
+action_msgs/GoalInfo goal
+---
+# All fields in response are user generated
```

Comment:
I guess `action_msgs/GoalInfo` should be added to the response so it can be associated with the request by the client.

Reply:
I was assuming the goal result service would rely on the service API to associate the request and response. Services have a sequence number, but you might be right that it's easier if the GoalInfo is in the result too.

Reply:
41d60e526865f03f4be8d93f18eb762022e45141